src/hooks/useAccountTotalStaked.js

```diff
@@ -1,31 +1,16 @@
 import { useMemo } from 'react'
+import { useAccountStakes } from './useStakes'
+
 import BigNumber from '../lib/bigNumber'
-import { addressesEqual } from '../lib/web3-utils'
-import { useWallet } from '../providers/Wallet'
-import { useAppState } from '../providers/AppState'
-import { PROPOSAL_STATUS_ACTIVE_STRING } from '../constants'
 
-export default function useAccountTotalStaked() {
-  const { proposals = [] } = useAppState()
-  const { account } = useWallet()
+export default function useAccountTotalStaked(account) {
+  const stakes = useAccountStakes(account)
 
-  const totalStaked = useMemo(
+  return useMemo(
     () =>
-      proposals
-        .filter(({ status }) => status === PROPOSAL_STATUS_ACTIVE_STRING)
-        .reduce((acc, { stakes }) => {
-          const myStake = stakes.find(({ entity }) =>
-            addressesEqual(entity, account)
-          )
-
-          if (!myStake) {
-            return acc
-          }
-
-          return acc.plus(myStake.amount)
-        }, new BigNumber(0)),
-    [proposals, account]
+      stakes.reduce((acc, { amount }) => {
```

Comment:
just to double check are we returning only the active ones on the useAccountStakes right?

Reply:
Yes! We are filtering by the actives ones only